traffic_ops/traffic_ops_golang/cdn/capacity.go

```diff
@@ -117,10 +119,22 @@ func getMonitorsCapacity(tx *sql.Tx, monitors map[tc.CDNName][]string) (Capacity
 	}, nil
 }
 
+func checkServiceInterface(tx *sql.Tx, name tc.InterfaceName, cacheName tc.CacheName) bool {
+	var result *bool
+	if err := tx.QueryRow(`SELECT i.service_address FROM ip_address AS i JOIN server as s on s.id = i.server WHERE i.interface=$1 AND s.host_name=$2`, name, cacheName).Scan(&result); err != nil {
+		log.Errorln("Error checking for service interface ", err)
+		return false
+	}
+	if *result == true {
+		return true
+	}
+	return false
```

Comment:
Since we're returning `true` when `*result` is `true` and `false` otherwise, this can just be

```golang
return *result
```